spec/std/uuid_spec.cr

```diff
@@ -269,4 +269,21 @@ describe "UUID" do
       UUID.v5_x500(data).v5?.should eq(true)
     end
   end
+
+  describe "v7" do
+    it "generates a v7 UUID" do
+      uuid = UUID.v7
+      uuid.v7?.should eq true
+      uuid.variant.rfc9562?.should eq true
+    end
+
+    it "generates UUIDs that are sortable with 1ms precision" do
+      uuids = Array.new(10) do
```

Comment:
`nitpick`: could be StaticArray

Reply:
IMO for specs this really doesn't matter. It's a one-time allocation.